src/components/header/modals/ModalAccountUnification.vue

```diff
@@ -61,13 +67,21 @@
         <au-step5
           :account-name="accountName"
           :selected-evm-address="selectedEvmAddress"
+          :avatar-url="avatar?.image ?? ''"
           :is-busy="isLoading"
           @next="updateSteps(6)"
         />
       </div>
       <div v-else-if="currentStep === 6">
         <au-step6 />
       </div>
+      <div v-else-if="currentStep === 200">
```

Comment:
Perhaps we might want to make a enum for the number of `currentStep` or make some comments.

Reply:
> Perhaps we might want to make a enum for the number of `currentStep` or make some comments.

I was thinking about that, but too lazy to implement

Reply:
Fixed in https://github.com/AstarNetwork/astar-apps/pull/975/commits/63a6481e3ee117d41c5c1b5cd1679a4d8b1d34a0